### DISCUSSION TITLE
Idea: autofill text

### DISCUSSION THREAD

**Discussion by YSubmarina** [SubtitleEdit/subtitleedit#5477] [Ideas]
To autofill words as we type, just like in Word. I translate subtitles manually and sometimes I miss the possibility to build our own dictionary of abbreviations to certain words that we have to write multiple times, to speed up the process a bit. 
You could check turn it on/off too.
I love SE, by the way!
Thank you :)

**Comment by darnn** [comment]
For the record, I use Breevy for this purpose:
http://www.16software.com/breevy/

As well as for de facto macros, i.e. I type something and SE performs "merge with next line", or, like, inserts a new line with a starting time and duration of zero. The first requires a keyboard shortcut, instead of just something you can type, and the second doesn't exist as a function at all, so this is what I use as a workaround. You could probably do something better with AHK but it's been too long since I've messed with it.
